x-pack/platform/plugins/shared/fleet/server/routes/agent_policy/handlers.ts

```diff
@@ -377,6 +377,15 @@ export const createAgentPolicyHandler: FleetRequestHandler<
       }
     }
 
+    if (
+      appContextService.getExperimentalFeatures().disableAgentlessLegacyAPI &&
```

Comment:
This can be considered a breaking change, could it cause an issue with existing users?

Reply:
Agentless is not GA yet, also creating agentless with agent/package policy API allow to create invalid policies, so I think we are fine. I was planning to communicate to our internal users before turning that flag.